### PR TITLE
add Reverie Labs to users

### PR DESCRIPTION
**Pull request by flaviuvadan**
@dmerrick I am wondering if you and Reverie Labs are open to listing the organization as users of Hera via `USERS.md`? 🙂 I think it would strengthen Hera's presence in the community and help with adoption!

### CODE REVIEW

**Comment by dmerrick** [comment]
Let me get back to you on this!

File: USERS.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ Please open a PR with your organization or project name in the following section
 
 1. [Dyno Therapeutics](https://www.dynotx.com)
 2. [Pipekit](https://www.pipekit.io)
+3. [ReverieLabs](https://www.reverielabs.com/)
```
Comment (dmerrick) [inline USERS.md:12]: please add a space between Reverie and Labs

**Review by dmerrick — CHANGES_REQUESTED** [review]
looks good after the space

File: USERS.md (hunk 1 above)
Comment (flaviuvadan) [reply USERS.md]: Thank you and Reverie Labs, @dmerrick! ❤️

**Pull request merged**